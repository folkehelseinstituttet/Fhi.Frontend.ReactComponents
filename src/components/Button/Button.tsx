import { FC } from 'react';

type Props = {
  text: string,
  onClick: () => void,
};

const Button: FC<Props> = ({ text, onClick }) => <button type="button" onClick={onClick}>{text}</button>;

export default Button;
