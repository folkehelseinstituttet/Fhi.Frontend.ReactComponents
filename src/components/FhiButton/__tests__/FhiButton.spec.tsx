import renderer from 'react-test-renderer';
import Button from '../FhiButton';

describe('FhiButton', () => {
  test('snapshot test', () => {
    const button = renderer.create(<Button onClick={() => {}}>test</Button>);
    const tree = button.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
