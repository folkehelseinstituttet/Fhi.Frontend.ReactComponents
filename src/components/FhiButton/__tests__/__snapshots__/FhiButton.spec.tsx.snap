// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FhiButton snapshot test 1`] = `
<button
  className=" btn fhi-btn-primary"
  onClick={[Function]}
  type="button"
>
  test
</button>
`;
